python/ray/tests/test_placement_group.py

```diff
@@ -1481,13 +1484,17 @@ def ping(self):
     ray.get(actor.ping.remote())
 
     # Create another placement group and make sure its creation will failed.
-    same_name_pg = ray.util.placement_group(
-        [{
-            "CPU": 1
-        } for _ in range(2)],
-        strategy="STRICT_SPREAD",
-        name=global_placement_group_name)
-    assert not same_name_pg.wait(10)
+    error_creation_count = 0
```

Comment:
Can you also verify if we have this test case? (Create placement group and remove it right after that. It should just remove properly as creation is sync).

Reply:
Yeah~ I will add this test.